packages/app-address-book/src/Forgetting.tsx

```diff
@@ -77,10 +76,13 @@ class Forgetting extends React.PureComponent<Props> {
           {t('Confirm address removal')}
         </Modal.Header>
         <Modal.Content className='forgetting-Address'>
-          <AddressSummary
-            className='ui--AddressSummary-base'
+          <AddressRow
+            isInline
             value={address || ''}
-          />
+          >
+            <p>{t('You are about to remove this address from your address book. Once completed, should you need access again, you need to re-add the address.')}</p>
```

Comment:
Same suggestion as before
```suggestion
            <p>{t('You are about to remove this address from your address book. Once completed, should you need to access it again, you will have to re-add the address.')}</p>
```